Billmike/Chessecake-Factory
# Designed Homepage

### What does this PR do?
Designs the homepage for the application.
### Description of tasks to be completed?
Design the Homepage of the application
### How should this be tested?
After cloning the repo, Open up ```index.html``` in your browser to see the homepage design.
